### PR TITLE
UMIP to add YAM collateral

### PR DESCRIPTION
**Pull request by rossgalloway**
Adding a UMIP for YAM collateral.  Sorry if I did something wrong.  First time pull requester!

### CODE REVIEW

File: UMIPs/umip-draft-yam-collateral.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## Headers
+| UMIP-38  |                                                                                                                                         |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------|                                                                                                                                    
+| UMIP Title | Add YAM as a collateral currency                  |
+| Authors    | Ross Galloway (ross@yam.finance)  |
+| Status     | Draft                                                                                    |                                               
+| Created    | January 23, 2021                                                                       |
```
Comment (smb2796) [inline UMIPs/umip-draft-yam-collateral.md:7]: Can you create an UMA Discourse discussion introducing this UMIP and add another row and add another row linking to that discussion? https://discourse.umaproject.org/

Comment (rossgalloway) [reply UMIPs/umip-draft-yam-collateral.md:8]: Forum post included now. That should have worked anyway.

**Review by smb2796 — COMMENTED** [review]
Looks great to me! Don't see any outstanding issues

**Comment by smb2796** [comment]
Should we be expecting a USDYAM price identifier proposal soon?

File: UMIPs/umip-draft-yam-collateral.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Security considerations
+
+Anyone deploying a new priceless token contract using this collateral should take note to parameterize the contract appropriately to the reference asset’s volatility and liquidity characteristics to avoid the loss of funds for synthetic token holders. Additionally, the contract deployer should ensure that there is a network of liquidators and disputers ready to perform the services necessary to keep the contract solvent. $UMA-holders should evaluate the ongoing cost and benefit of supporting this asset as collateral and also contemplate removing support of this collateral if liquidity concerns are identified. $UMA-holders should take note of the collateral’s nature as liquidity of the collateral changes, or if added robustness (eg via TWAPs) are necessary to prevent market manipulation.
+
+YAM itself should present  little risk. It is a standard ERC-20 token. The rebasing function of the token contract was turned off on December 26, 2020 (https://etherscan.io/tx/0xf7b24fc7b69b6b85f28089691406ba57d878a799f0753d2f6a9f2d5821b77c9a) so there are no longer complications that would arise from a dynamically
```
[the remark this answers is not in the document]
Comment (smb2796) [reply UMIPs/umip-draft-yam-collateral.md:47]: This is a good point, and something we should keep in mind for all collateral types

**Review by smb2796 — APPROVED** [review]
Going to merge in and change the name/status!

**Pull request merged**